migrate/v0_11/migrate.go

```diff
@@ -57,11 +57,92 @@ func MigrateBep3(oldGenState v0_9bep3.GenesisState) v0_11bep3.GenesisState {
 		}
 		swaps = append(swaps, newSwap)
 	}
+
+	// -------------- ADD BTCB To BEP3 params --------------------
+	btcbAssetParam := v0_11bep3.NewAssetParam(
+		"btcb",
+		0,
+		v0_11bep3.SupplyLimit{
+			Limit:          sdk.NewInt(10000000000), // 100 BTC limit at launch
+			TimeLimited:    false,
+			TimePeriod:     time.Duration(0),
+			TimeBasedLimit: sdk.ZeroInt()},
+		true,
+		mustAccAddressFromBech32("kava14qsmvzprqvhwmgql9fr0u3zv9n2qla8zhnm5pc"),
+		v0_9Params.BnbDeputyFixedFee,
+		sdk.OneInt(),
+		sdk.NewInt(1000000000),
+		220,
+		270,
+	)
+	btcbAssetSupply := v0_11bep3.NewAssetSupply(
+		sdk.NewCoin("btcb", sdk.ZeroInt()),
+		sdk.NewCoin("btcb", sdk.ZeroInt()),
+		sdk.NewCoin("btcb", sdk.ZeroInt()),
+		sdk.NewCoin("btcb", sdk.ZeroInt()),
+		time.Duration(0))
+	assetParams = append(assetParams, btcbAssetParam)
+	assetSupplies = append(assetSupplies, btcbAssetSupply)
+	xrpbAssetParam := v0_11bep3.NewAssetParam(
+		"xrpb",
+		144,
+		v0_11bep3.SupplyLimit{
+			Limit:          sdk.NewInt(1000000000000), // 1,000,000 XRP limit at launch
```

Comment:
1 million XRP is worth $240,000 while 100 BTC is worth ~$1 million USD. This variance is very likely intentional (we're picking round numbers for asset amounts that are generally in the same USD ballpark?) but wanted to mention the difference.

Reply:
Yes, this was intentional. The stability committee can adjust limits pretty rapidly, so these values are less important to get exactly correct.